<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenerateBlocksDlg</class>
 <widget class="QWidget" name="GenerateBlocksDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GenerateBlocksDlg</string>
  </property>
  <widget class="QLineEdit" name="lineEditOutputDirectory">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Output directory:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNumScenarios">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string># Scenarios:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOutputDirectory">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>50</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditBuildingFootprintsImageSize">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>410</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>450</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>521</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Parcels</string>
   </property>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Parcel area:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParcelAreaMin">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParcelAreaMax">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>226</x>
      <y>30</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>~</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Setback front:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSetbackFrontMin">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSetbackFrontMax">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>226</x>
      <y>60</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>~</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Setback rear:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSetbackRearMin">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSetbackRearMax">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>226</x>
      <y>90</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>~</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Setback side:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSetbackSideMin">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSetbackSideMax">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>226</x>
      <y>120</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>~</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_38">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Std Dev:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditParcelAreaDev">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>[m]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>[m]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>[m]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>[m^2]</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_40">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Block footprints image size</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>450</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>521</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Blocks</string>
   </property>
   <widget class="QLabel" name="label_35">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Block width:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBlocksWidthMin">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBlocksWidthMax">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_39">
    <property name="geometry">
     <rect>
      <x>226</x>
      <y>20</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>~</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_41">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Block height:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBlocksHeightMin">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBlocksHeightMax">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_42">
    <property name="geometry">
     <rect>
      <x>226</x>
      <y>50</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>~</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>[m]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>[m]</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEditBatchNum">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_43">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Batch number:</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
